### DISCUSSION TITLE
What is the dataset format would I set? JSON format or .bin and .idx format(convert by Megatron-LM process_data.py)

### DISCUSSION THREAD

**Discussion by 13416157913** [alibaba/Megatron-LLaMA#2] [Q&A]
Hello I want to ask，
What is the dataset format would I set? JSON format or .bin and .idx format(convert by Megatron-LM process_data.py)？
Thanks  a lot.

**Comment by li-yi-dong** [comment]
Replied in the issue https://github.com/alibaba/Megatron-LLaMA/issues/3
